<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>urpmi</class>
 <widget class="QDialog" name="urpmi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>614</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mantenimiento de repositorios</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recoverdrake.qrc">
    <normaloff>:/Imagenes/rpm.png</normaloff>:/Imagenes/rpm.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>671</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Visor de registros de la base de datos.</string>
     </property>
     <property name="statusTip">
      <string>Visor de registros de la base de datos.</string>
     </property>
     <property name="whatsThis">
      <string>Visor de registros de la base de datos.</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoScrollMargin">
      <number>26</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="recoverdrake.qrc">
        <normaloff>:/Imagenes/good.png</normaloff>:/Imagenes/good.png</iconset>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Repositorio</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Activado</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Desactivado</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Actualizable</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="3" rowspan="3">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Acceso global</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_14">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Comprobacion de todos los repositorios disponibles.</string>
          </property>
          <property name="statusTip">
           <string>Comprobacion de todos los repositorios disponibles.</string>
          </property>
          <property name="whatsThis">
           <string>Comprobacion de todos los repositorios disponibles.</string>
          </property>
          <property name="text">
           <string>Verificar repositorios</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/lupaO.png</normaloff>:/Imagenes/lupaO.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Eliminacion de todos los repositorios disponibles.</string>
          </property>
          <property name="statusTip">
           <string>Eliminacion de todos los repositorios disponibles.</string>
          </property>
          <property name="whatsThis">
           <string>Eliminacion de todos los repositorios disponibles.</string>
          </property>
          <property name="accessibleName">
           <string>Eliminacion de todos los repositorios disponibles.</string>
          </property>
          <property name="text">
           <string>Eliminar todos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_6">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Seleccion masiva de todos los paquetes.</string>
          </property>
          <property name="statusTip">
           <string>Seleccion masiva de todos los paquetes.</string>
          </property>
          <property name="whatsThis">
           <string>Seleccion masiva de todos los paquetes.</string>
          </property>
          <property name="text">
           <string>Marcar todos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/good.png</normaloff>:/Imagenes/good.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_7">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Deseleccion masiva de todos los paquetes.</string>
          </property>
          <property name="statusTip">
           <string>Deseleccion masiva de todos los paquetes.</string>
          </property>
          <property name="whatsThis">
           <string>Deseleccion masiva de todos los paquetes.</string>
          </property>
          <property name="text">
           <string>Demarcar todos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/stop.png</normaloff>:/Imagenes/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>inserccion de todos los repositorios posibles de instalacion.</string>
          </property>
          <property name="statusTip">
           <string>inserccion de todos los repositorios posibles de instalacion.</string>
          </property>
          <property name="whatsThis">
           <string>inserccion de todos los repositorios posibles de instalacion.</string>
          </property>
          <property name="accessibleName">
           <string>inserccion de todos los repositorios posibles de instalacion.</string>
          </property>
          <property name="text">
           <string>Anadir todos</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/Correcto.png</normaloff>:/Imagenes/Correcto.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Acceso individual</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_10">
          <property name="toolTip">
           <string>Marca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="statusTip">
           <string>Marca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="whatsThis">
           <string>Marca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="text">
           <string>Activar seleccionado</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/good.png</normaloff>:/Imagenes/good.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_12">
          <property name="toolTip">
           <string>Desmarca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="statusTip">
           <string>Desmarca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="whatsThis">
           <string>Desmarca la linea activa de la base de datos para su tratamiento.</string>
          </property>
          <property name="text">
           <string>Desactivar seleccionado</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/stop.png</normaloff>:/Imagenes/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_11">
          <property name="toolTip">
           <string>Elimina todos los registros seleccionados.</string>
          </property>
          <property name="statusTip">
           <string>Elimina todos los registros seleccionados.</string>
          </property>
          <property name="whatsThis">
           <string>Elimina todos los registros seleccionados.</string>
          </property>
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
          <property name="icon">
           <iconset resource="recoverdrake.qrc">
            <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>465</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Total Repositorios:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Numero de registros activos en la base de datos activa.</string>
     </property>
     <property name="statusTip">
      <string>Numero de registros activos en la base de datos activa.</string>
     </property>
     <property name="whatsThis">
      <string>Numero de registros activos en la base de datos activa.</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="toolTip">
      <string>Opciones de urpmi que pueden ser elegidas entre las mostradas.</string>
     </property>
     <property name="statusTip">
      <string>Opciones de urpmi que pueden ser elegidas entre las mostradas.</string>
     </property>
     <property name="whatsThis">
      <string>Opciones de urpmi que pueden ser elegidas entre las mostradas.</string>
     </property>
     <property name="title">
      <string>Repositorios</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mandriva:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>Oficiales</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string>PLF</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox_7">
        <property name="text">
         <string>MIB</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string>BDK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="checkBox_9">
        <property name="text">
         <string>MUD</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mageia:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_10">
        <property name="text">
         <string>Oficiales</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_11">
        <property name="text">
         <string>BDK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Nimbus Sans L Condensed</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="toolTip">
      <string>Consola de procesos, donde se detalla al máximo lo que va procesando.</string>
     </property>
     <property name="statusTip">
      <string>Consola de procesos, donde se detalla al máximo lo que va procesando.</string>
     </property>
     <property name="whatsThis">
      <string>Consola de procesos, donde se detalla al máximo lo que va procesando.</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Nimbus Sans L Condensed'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:7pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QProgressBar" name="progressBar">
       <property name="toolTip">
        <string>Ratio de procesos realizados.</string>
       </property>
       <property name="statusTip">
        <string>Ratio de procesos realizados.</string>
       </property>
       <property name="whatsThis">
        <string>Ratio de procesos realizados.</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="toolTip">
        <string>Nos vamos..</string>
       </property>
       <property name="statusTip">
        <string>Nos vamos..</string>
       </property>
       <property name="whatsThis">
        <string>Nos vamos..</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
       <property name="icon">
        <iconset resource="recoverdrake.qrc">
         <normaloff>:/Imagenes/puerta.png</normaloff>:/Imagenes/puerta.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recoverdrake.qrc"/>
 </resources>
 <connections/>
</ui>
